test/e2e/instance-create.e2e.ts

```diff
@@ -7,7 +7,32 @@
  */
 import { floatingIp, images } from '@oxide/api-mocks'
 
-import { expect, expectNotVisible, expectRowVisible, expectVisible, test } from './utils'
+import {
+  expect,
+  expectNotVisible,
+  expectRowVisible,
+  expectVisible,
+  test,
+  type Page,
+} from './utils'
+
+const selectASiloImage = async (page: Page, name: string) => {
+  await page.getByRole('tab', { name: 'Silo images' }).click()
+  await page.getByRole('button', { name: 'Select an image' }).click()
+  await page.getByRole('option', { name }).click()
+}
+
+const selectAProjectImage = async (page: Page, index: number) => {
+  await page.getByRole('tab', { name: 'Project images' }).click()
+  await page.getByRole('button', { name: 'Image' }).click()
+  await page.getByRole('option', { name: images[index].name }).click()
+}
```

Comment:
How about changing this one to take a name like the others? That would make the call sites for this function more legible, less abstract. I think the fact that we were using indexes before was kinda wrong.